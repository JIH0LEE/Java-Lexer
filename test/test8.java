int func(int name,String str){
    if(true){
        return 0;
    }
    else{
        return 1;
    }
}
