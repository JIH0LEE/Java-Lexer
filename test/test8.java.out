<1,VTYPE,int>
<1,ID,func>
<1,LPAREN,(>
<1,VTYPE,int>
<1,ID,name>
<1,COMMA,,>
<1,VTYPE,String>
<1,ID,str>
<1,RPAREN,)>
<1,LBRACE,{>
<2,IF,if>
<2,LPAREN,(>
<2,BOOLSTR,true>
<2,RPAREN,)>
<2,LBRACE,{>
<3,RETURN,return>
<3,NUM,0>
<3,SEMI,;>
<4,RBRACE,}>
<5,ELSE,else>
<5,LBRACE,{>
<6,RETURN,return>
<6,NUM,1>
<6,SEMI,;>
<7,RBRACE,}>
<8,RBRACE,}>
